workspaces/marketplace/plugins/marketplace-common/src/api/MarketplaceBackendClient.ts

```diff
@@ -141,6 +141,15 @@ export class MarketplaceBackendClient implements MarketplaceApi {
     );
   }
 
+  async getPackageConfigByName(
+    namespace: string,
+    name: string,
+  ): Promise<{ configYaml: string }> {
```

Comment:
Minor: You can define type instead of `{ configYaml: string }`, but it is up to you.